torch/_torch_docs.py

```diff
@@ -5182,13 +5182,24 @@ def merge_dicts(*dicts):
     >>> torch.quantile(a, q)
     tensor([-0.5446,  0.0700,  0.9214])
 
-.. function:: quantile(input, q, dim=None, keepdim=False, *, out=None) -> Tensor
+.. function:: quantile(input, q, dim=None, interpolation='linear', keepdim=False, *, out=None) -> Tensor
 
-Returns the q-th quantiles of each row of the :attr:`input` tensor along the dimension
-:attr:`dim`, doing a linear interpolation when the q-th quantile lies between two
-data points. By default, :attr:`dim` is ``None`` resulting in the :attr:`input` tensor
+Returns the q-th quantiles of each row of the :attr:`input` tensor
+along the dimension :attr:`dim` based on :attr:`interpolation`.
+When the desired quantile lies between two data points ``i < j``,
+the result is computed based on the :attr:`interpolation` value as described below.
+By default, :attr:`interpolation` is ``linear`` and :attr:`dim` is ``None`` resulting in the :attr:`input` tensor
 being flattened before computation.
 
+When the quantile value lies between two data points ``i < j``,
+the result is computed according to the given :attr:`interpolation` method as follows:
+
+- ``linear``: ``i + (j - i) * fraction``, where ``fraction`` is the fractional part of the index surrounded by ``i`` and ``j``.
```

Comment:
"the fractional part of the index surrounded by" is too vague. `fraction` could be replaced with a function of `q` like `q % 1`. Maybe there's a more common way to express that only the decimal or "fractional" part of a number (in base 10) is desired?